src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -781,7 +782,7 @@ static void window_guest_list_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi,
                                 gfx_draw_sprite(dpi, STR_ENTER_SELECTION_SIZE, 112, y + 1, 0);
 
                             // Action
-                            peep->FormatActionTo(gCommonFormatArgs);
+                            peep->FormatActionTo(ft);
```

Comment:
```suggestion
                            ft = Formatter::Common();
                            peep->FormatActionTo(ft);
```
I think this needs resetting to the empty Common too